AppInspector.CLI/Writers/AnalyzeHtmlWriter.cs

```diff
@@ -107,6 +114,39 @@ private void WriteJsonResult()
             WriteOnce.Verbosity = saveVerbosity;
         }
 
+
+
+        private string MergeResourceFiles(string inputPath)
+        {
+            StringBuilder stringBuilder = new StringBuilder();
+
+            if (Directory.Exists(inputPath))
+            {
+                try
+                {
+                    IEnumerable<string> srcfileList = Directory.EnumerateFiles(inputPath, "*.*", SearchOption.AllDirectories);
+                    if (srcfileList.Count() == 0)
```

Comment:
Use .Any

Reply:
(Advantage of .Any() is that it stops after the first match, so for this, it should be *much* faster.)

Reply:
Yeah good call.  The function where this will be used is temporarily removed but will be reintroduced as a separate PR focused on resource output consolidation and I'll use that approach for similar needs.